modules/storage/postgresql-single-server/README.md

```diff
@@ -0,0 +1,159 @@
+| `sourceServerResourceId`         | `string`       | No       | The source server id to restore from. Leave empty if creating from scratch.                                                                                                                                                                                                                                                                                                                  |
+| `sslEnforcement`                 | `bool`         | No       | Toggle SSL enforcement for incoming connections.                                                                                                                                                                                                                                                                                                                                             |
+| `storageAutogrow`                | `bool`         | No       | Toggle storage autogrow.                                                                                                                                                                                                                                                                                                                                                                     |
+| `roleAssignments`                | `array`        | No       | Array of role assignment objects that contain the "roleDefinitionIdOrName" and "principalId" to define RBAC role assignments on this resource. In the roleDefinitionIdOrName attribute, provide either the display name of the role definition, or its fully qualified ID in the following format: "/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11" |
+
+## Outputs
+
+| Name              | Type   | Description                                                        |
+| :---------------- | :----: | :----------------------------------------------------------------- |
+| resourceGroupName | string | The resource group the Postgresql Single Server was deployed into. |
+| fqdn              | string | FQDN of the generated Postgresql Single Server                     |
+| id                | string | The resource ID of the Postgresql Single Server.                   |
+
+## Examples
+
+### Example 1
+
+Deploy a Postgresql Single Server with minimal parameters
+
+```bicep
+@secure
+param administratorLoginPassword string
+
+module postgresqlSingleServer 'br/public:storage/postgresql-single-server:1.0' = {
```

Comment:
The module will have version `1.0.1` once published. Can you update the version number for the examples?